makelib/helm.mk

```diff
@@ -92,6 +92,7 @@ helm.prepare: helm.prepare.$(1)
 
 helm.lint.$(1): $(HELM_HOME) helm.prepare.$(1)
 	@rm -rf $(abspath $(HELM_CHARTS_DIR)/$(1)/charts)
+	@$(HELM) dependency update $(abspath $(HELM_CHARTS_DIR)/$(1))
```

Comment:
Just noting from our conversation that the reason that we are not gating this on Helm 3 is because you should theoretically want to lint with your dependencies present anyway as they will eventually get pulled in during build (`helm.dep` is dependency of `build.check`).